### ISSUE TITLE
TechDocs: Remove "Made with Material for MkDocs" from documentation pages

### ISSUE THREAD

**Issue by OrkoHunter** [spotify/backstage#2850]
## Feature Suggestion

On TechDocs pages, MkDocs inserts a "Made with Material for MkDocs" on every page. This is done when the pages are being generated.

## Possible Implementation

Use a [transformer](https://backstage.io/docs/features/techdocs/concepts) in TechDocs reader to remove this once it is generated. Look for inspiration in ["Remove mkdocs header transformer"](https://github.com/spotify/backstage/blob/master/plugins/techdocs/src/reader/transformers/removeMkdocsHeader.ts).

## Context

![Screenshot 2020-10-12 at 10 06 40](https://user-images.githubusercontent.com/8065913/95721488-f28c0600-0c72-11eb-9722-52d84db8f7fc.png)

**Comment by themousepotato** [comment]
Looks pretty much straightforward. Can I work on this?

**Comment by OrkoHunter** [comment]
Absolutely! ❤️

**Comment by divz11** [comment]
Hi Sir, I've completed this (My first Open-Source Contribution).

**Comment by divz11** [comment]
Hey @OrkoHunter, I have removed those lines from the file.
Please accept it.

**Comment by divz11** [comment]
Hey @OrkoHunter, I have removed that component please Accept my PR #https://github.com/spotify/backstage/pull/2984/commits/211e085f8eb6553fdf43c51543b20fa8a6f06cd6

**Comment by themousepotato** [comment]
I was working on this but I couldn't reproduce the issue because my techdocs wasn't loading at all. @divz11 Were you able to reproduce it? I had talked with @hooloovooo on discord and we couldn't get it solved.

**Comment by emmaindal** [comment]
@themousepotato Hey, are you still seeing issues with techdocs not loading? What issues do you see?

**Comment by themousepotato** [comment]
Hey @emmaindal, I used to get a 404. I'd posted the logs in the discord channel a few days before. Now, I'm getting the following error:
```
TypeError: Invalid type in config for key 'integrations.github' in 'app-config.yaml', got array, wanted object at ConfigReader.readValue (webpack-internal:///../config/src/reader.ts:321:15) at ConfigReader.readConfigValue (webpack-internal:///../config/src/reader.ts:276:24) at eval (webpack-internal:///../config/src/reader.ts:7:691) at eval (webpack-internal:///../config/src/reader.ts:279:128) at _optionalChain (webpack-internal:///../config/src/reader.ts:7:669) at ConfigReader.readConfigValue (webpack-internal:///../config/src/reader.ts:279:14) at ConfigReader.getOptionalString (webpack-internal:///../config/src/reader.ts:237:17) at getGithubApiUrl (webpack-internal:///../../plugins/techdocs-backend/src/default-branch.ts:57:12) at getGithubDefaultBranch (webpack-internal:///../../plugins/techdocs-backend/src/default-branch.ts:138:16) at getDefaultBranch (webpack-internal:///../../plugins/techdocs-backend/src/default-branch.ts:242:22)
```

**Comment by OrkoHunter** [comment]
@themousepotato The fix for the above issue has been merged. Could you rebase and try again? Thanks!

**Comment by themousepotato** [comment]
Doing that right away! I'll let you know how it goes.

**Comment by themousepotato** [comment]
Hey @OrkoHunter. It's still giving 404. I think it's because the cloning is taking way too much time than usual. Is there anything can be done to replace cloning from scratch and make it use of the clone I have locally?

**Comment by emmaindal** [comment]
closed by #3072
